pkgs/by-name/up/upbound-main/package.nix

```diff
@@ -0,0 +1,5 @@
+{ callPackage }:
+
+callPackage ../upbound/package-common.nix {
+  version-channel = "main";
+}
```

Comment:
```suggestion
{ upbound }:

upbound.override {
  version-channel = "main";
}
```

Reply:
Packages in pkgs/by-name must be isolated, they are not allowed to refer to outside files.